R/run_trisk.R

```diff
@@ -150,7 +154,7 @@ run_stress_test_iteration <- function(args_list) {
 }
 
 # Avoid R CMD check NOTE: "Undefined global functions or variables"
-globalVariables(c(names(formals(run_trisk)), "iter_var"))
+globalVariables(c(names(formals(run_trisk)), "iter_var", "end_year"))
```

Comment:
why did you do this here and not for lrisk?

Reply:
adding it - i starting doing it bc of a note that was appearing which vanished once i had done it for trisk